### PR TITLE
Pull Request to merge my blog on UserStories 

### PR DESCRIPTION
**Pull request by Shikha-Nandal**
Please add a direct link to your post here:

https://taddison-scottlogic.github.io/blog/2023/08/18/the-power-of-a-well-written-user-story.html

Have you (please tick each box to show completion):

 - [X] Added your blog post to a single category?
 - [X] Added a brief summary for your post? Summaries should be roughly two sentences in length and give potential readers a good idea of the contents of your post.
 - [X] Checked that the build passes?
 - [X] Checked your spelling (you can use `npm spellcheck` if that's your thing)
 - [X] Ensured that your author profile contains a profile image, and a brief description of yourself? (make it more interesting than just your job title!)
 - [X] Optimised any images in your post? They should be less than 100KBytes as a general guide.

Posts are approved based on their category. replace `(at)` with `@` to notify them!

 - Tech - Colin Eberhardt - (at)ColinEberhardt
 - Cloud - Colin Eberhardt - (at)ColinEberhardt
 - Delivery - Matt Phillips - (at)scottlogicmphillips
 - UX - Graham Odds - (at)godds
 - People - Graham Odds - (at)godds
 - Testing - Laurence Pisani - @lpisani-SL

### CODE REVIEW

File: _posts/2023-08-18-the-power-of-a-well-written-user-story.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+**Definition of Done:** The conditions that must be met for the *User Story* to be considered complete, including aspects of design, testing, and documentation.
+
+**Acceptance Sign-Off:** People/Team who will approve the acceptance criteria and sign off on the completion of the *User Story*.
+
+
+Additionally, it is important to keep the *User Story* updated with any significant changes in any of the above components.
+
+An efficient way of making sure that the *User Stories* do not miss any important information is to use an internally agreed upon *User Story* template. Not Every *User Story* would need each one of the above components; however, having added these in the template would act as a checklist and ensure that the information will not me missed. 
+
+**As a test analyst, it is important to understand the critical role that *User Stories* play in guiding the testing process and ensuring the delivery of high-quality software. And hence we need to ensure that each *User Story* is capable of not only meeting its acceptance criteria but also contributing to the overall excellence of the product.**
```
Comment (lpisani-SL) [inline _posts/2023-08-18-the-power-of-a-well-written-user-story.md]: Test Analyst is not a recognised role at Scott Logic, we refer to Test Engineers or Test Consultants

**Review by lpisani-SL — COMMENTED** [review]
Just need to attend to the reference to Test Analysts... As this is not recognised role at Scott Logic we should be referencing Test Engineers or Test Consultants instead to ensure consistency

File: _posts/2023-08-18-the-power-of-a-well-written-user-story.md (hunk 1 above)
Comment (Shikha-Nandal) [reply _posts/2023-08-18-the-power-of-a-well-written-user-story.md]: Replaced "Test Analyst" with "Test Engineer"

**Review by lpisani-SL — APPROVED** [review]

**Comment by taddison-scottlogic** [comment]
Post name updated to:
https://taddison-scottlogic.github.io/blog/2023/09/11/the-power-of-a-well-written-user-story.html

**Comment by ColinEberhardt** [comment]
Thank you @taddison-scottlogic  and @Shikha-Nandal

**Pull request merged**